Makefile

```diff
@@ -662,7 +659,7 @@ endif
 ifeq ($(KBUILD_EXTMOD),)
 # Objects we will link into vmlinux / subdirs we need to visit
 init-y		:= init/
-drivers-y	:= drivers/ sound/ ubuntu/
+drivers-y	:= drivers/ sound/
```

Comment:
Not necessarily for this review since it'd be nice to keep all of the kernels in sync as much as possible, but I'm noticing now that `sound` is still included here. It feels like something that we could also do without. You do turn off `CONFIG_SOUND`, but it looks like the `sound/Makefile` also includes things like `CONFIG_DMASOUND`, `CONFIG_SND`, `CONFIG_SND_AOA`, `CONFIG_AC97_BUS`, and `CONFIG_AC97_BUS_NEW`.